src/test/java/io/supertokens/test/Utils.java

```diff
@@ -31,22 +32,21 @@
 
 import java.io.*;
 import java.nio.charset.StandardCharsets;
```

Comment:
Changes to this file should also be reflected in the plugin repos for the core.

Reply:
Although I changed the implementations of these methods to make them more readable, the method signatures did not change. No changes should be needed in the plugin repos.